test/base-stack.test.ts

```diff
@@ -0,0 +1,253 @@
+import path from "path";
```

Comment:
I assume you wish to add assertions to these tests later on?

Reply:
Added in commit:  https://github.com/awslabs/aws-ddk/pull/254/commits/4ff3cdd2531a1c86a1d3000680926e7fcf3a46a2